style/sass/sample.scss

```diff
@@ -1,53 +1,35 @@
-@import "bourbon";
+@import "partial-name";
```

Comment:
I wanted to show a two-word name, to indicate the use of hyphens. Sass’s documentation uses the term “partial”, so I wanted to maintain that clarity.